<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fractal Image Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>593</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,5">
    <item>
     <widget class="RenderArea" name="renderArea" native="true"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>X Scale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="in_scale_x"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Y Scale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="in_scale_y"/>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_auto_scale">
            <property name="text">
             <string>1:1 (Ignore X scale)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Const: Re</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="in_const_x"/>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Const: Im</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="in_const_y"/>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Max Iterations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="in_max_depth"/>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&quot;Infinity&quot; Threshold</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="in_mod_threshold"/>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QDial" name="k_scale_x">
          <property name="minimum">
           <number>500</number>
          </property>
          <property name="maximum">
           <number>8000</number>
          </property>
          <property name="singleStep">
           <number>50</number>
          </property>
          <property name="pageStep">
           <number>50</number>
          </property>
          <property name="value">
           <number>2000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDial" name="k_scale_y">
          <property name="minimum">
           <number>500</number>
          </property>
          <property name="maximum">
           <number>8000</number>
          </property>
          <property name="singleStep">
           <number>50</number>
          </property>
          <property name="pageStep">
           <number>50</number>
          </property>
          <property name="value">
           <number>2000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDial" name="k_const_x">
          <property name="minimum">
           <number>-600</number>
          </property>
          <property name="maximum">
           <number>600</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>370</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDial" name="k_const_y">
          <property name="minimum">
           <number>-600</number>
          </property>
          <property name="maximum">
           <number>600</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>-300</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDial" name="k_max_depth">
          <property name="minimum">
           <number>16000</number>
          </property>
          <property name="maximum">
           <number>128000</number>
          </property>
          <property name="singleStep">
           <number>1000</number>
          </property>
          <property name="pageStep">
           <number>1000</number>
          </property>
          <property name="value">
           <number>32000</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QDial" name="k_mod_threshold">
          <property name="minimum">
           <number>500</number>
          </property>
          <property name="maximum">
           <number>64000</number>
          </property>
          <property name="singleStep">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>200</number>
          </property>
          <property name="value">
           <number>4000</number>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,5">
        <item>
         <widget class="QPushButton" name="btn_update">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_autoPaint">
          <property name="text">
           <string>Auto</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_save_as"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="act_save_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderArea</class>
   <extends>QWidget</extends>
   <header>renderarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
